helm/README.md

```diff
@@ -1,15 +1,79 @@
-# THIS CHART IS DEPRECATED
-The chart has been replaced by [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) and is no longer maintained. Therefore we are currently not accepting any pull requests. Thank you for your understanding.
+# Moved
+The helm charts originally developed as part of this repository have been moved to [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator).
+
+The multiple charts built in this repository have been combined into a single chart that installs prometheus operator, prometheus, alertmanager, grafana as well as the multitude of exporters necessary to monitor a cluster.
+
+There is no direct migration path from this chart to the stable/prometheus-operator chart - there are numerous changes and capability enhancements.
+
+It is still possible to run multiple prometheus instances on a single cluster - you will need to disable the parts of the chart you do not wish to deploy.
+
+Issues and pull requests should be tracked using the [helm/charts](https://github.com/helm/charts) repository.
 
 You can check out the tickets for this change [here](https://github.com/coreos/prometheus-operator/issues/592) and [here](https://github.com/helm/charts/pull/6765)
 
----
+## Changes
+The chart has 3 dependencies, that can be seen in the chart's requirements file:
+https://github.com/helm/charts/blob/master/stable/prometheus-operator/requirements.yaml
```

Comment:
I understand we want to document the upstream helm chart changes but I am a bit worried this will skew over time. Can we maybe point the upstream chart's README instead?

Reply:
Thanks for the suggestion. I've moved the migration section to that chart and updated the link